### ISSUE TITLE
[Help]: Inquiry About Padding Handling in VALL-E AR Loss Calculation

### ISSUE THREAD

**Issue by jubang0219** [open-mmlab/Amphion#381]
Hi there, 

I have a question regarding the handling of padding during loss calculation in the AR training process of the VALL-E model.
* Relevant code link: https://github.com/open-mmlab/Amphion/blob/main/models/tts/valle/valle.py#L374

In the script Amphion/models/tts/valle/valle.py, is the padding not specifically handled during AR stage training?
>  ar_loss = F.cross_entropy(logits, targets, reduction=reduction)

For the NAR stage, I observed that the padding is processed using the ignore option.
> nar_loss = F.cross_entropy(
            logits,
            targets,
            ignore_index=self.audio_token_num,
            reduction=reduction,
        ) * (total_length / (total_length - prefix_len * x.shape[0]))

Thank you.
